apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -151,6 +151,20 @@ type SingleStepInstrumentation struct {
 	// ex: "java": "v1.18.0"
 	// +optional
 	LibVersions map[string]string `json:"libVersions,omitempty"`
+
+	// Language detection currently only detects languages and adds them as annotations on deployments, but doesn't use these languages for injecting libraries to applicative pods.
+	// It requires Agent 7.52+ and Cluster Agent 7.52+
```

Comment:
```suggestion
	// (Requires Agent 7.52.0+ and Cluster Agent 7.52.0+)
```
To match the existing descriptions that have `Requires Agent xxx`